target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -18,6 +18,7 @@ board_config_update
 
 case $board in
 11acnas|\
+afoundry-ew1200|\
```

Comment:
PLease remove the ```afoundry-``` prefix from all names (here, the model name in the dts, the led names and so on).